File: 22.09/168.go
func convertToTitle(columnNumber int) string {
    ans := ""
    for columnNumber > 26 {
        if columnNumber % 26 == 0 {
            ans = string(90) + ans
            columnNumber = columnNumber / 26 - 1
        } else {
            ans = string(64 + columnNumber % 26) + ans
            columnNumber = columnNumber / 26
        }
    }
    if columnNumber % 26 == 0 {
        ans = string(90) + ans
    } else {
        ans = string(64 + columnNumber % 26) + ans
    }
    
    return ans
}

File: typescript/23.08/168.ts
function convertToTitle(columnNumber: number): string {
    let ans = ""
    while (columnNumber > 26) {
        ans = String.fromCharCode(65 + (columnNumber-1)%26) + ans
        columnNumber = Math.floor((columnNumber-1)/26)
    }
    ans = String.fromCharCode(65 + (columnNumber-1)%26) + ans
    return ans
};
